RecaptchaValidation/Models/RecaptchaRequestMessage.cs

```diff
@@ -4,24 +4,25 @@
 
 namespace RecaptchaValidation.Models
 {
-    public class RecaptchaRequestMessage : IRecaptchaRequestMessage
+    public sealed class RecaptchaRequestMessage
     {
         public string path { get; private set; }
```

Comment:
properties in classes typically use PascalCase. so these could be:
Path
Secret
Response
RemoteIp